app/assets/javascripts/angular/controllers/orders_controller.js

```diff
@@ -2,25 +2,78 @@
 var app = angular.module('taxi', ['ngRoute']);
 
 app.controller('CreateOrderController', ['$scope', '$http', function($scope, $http) {
-    $scope.phone_pattern = /(0)[0-9]{9}/;
-    $scope.email_pattern = /^(.+)@(.+)$/;
-    $scope.addOrder = function() {
+  $scope.data = {
+    availableOptions: [
+      {id: '1'}, {id: '2'}, {id: '3'}, {id: '4'},
```

Comment:
@yelizaveta-dziuba, please use more informative hash keys. I think there should be something like `quantity`, not `id`, as far as it's related to passengers' quantity :sweat_smile: